docsrc/Managing_resources.rst

```diff
@@ -17,80 +17,118 @@ To get started, follow the steps below:
+    +---------------------+-----------------------------------------------------------------------------------------------------------------------------+
+    | ``server``          |  The API hostname for logging in. Defaults to ``api.app.firebolt.io`` if not included.                                      |
+    +---------------------+-----------------------------------------------------------------------------------------------------------------------------+
+    | ``default_region``  |  The default region for creating new databases and engines.                                                                 |
+    |                     |                                                                                                                             |
+    |                     |  For more information, see `Available AWS Regions <https://docs.firebolt.io/general-reference/available-regions.html>`_.    |
+    +---------------------+-----------------------------------------------------------------------------------------------------------------------------+
 
 
 
-	A ``Settings`` object can be configured with parameters by multiple methods.
+    A ``Settings`` object can be configured with parameters by multiple methods.
 
-		* Add the parameters manually in your command script:
+        * Add the parameters manually in your command script:
 
-			::
+            ::
 
-				settings = Settings(
-				    user="your_username",
-				    password="your_password",
-				    server="api.app.firebolt.io"
-				    default_region="your_region"
-				)
+                settings = Settings(
+                    user="your_username",
+                    password="your_password",
+                    server="api.app.firebolt.io"
+                    default_region="your_region"
+                )
 
-		* Use a ``.env`` file located in your root directory containing the following parameters:
+        * Use a ``.env`` file located in your root directory containing the following parameters:
 
-			::
+            ::
 
-				FIREBOLT_USER="your_username",
-				FIREBOLT_PASSWORD="your_password",
-				FIREBOLT_SERVER="api.app.firebolt.io"
-				FIREBOLT_DEFAULT_REGION="your_region"
+                FIREBOLT_USER="your_username",
+                FIREBOLT_PASSWORD="your_password",
+                FIREBOLT_SERVER="api.app.firebolt.io"
+                FIREBOLT_DEFAULT_REGION="your_region"
 
-			In your application file, the ``Settings`` object can read the values from the
-			``.env`` file if it is set to ``None`` instead of having values, as shown below:
+            In your application file, the ``Settings`` object can read the values from the
+            ``.env`` file if it is set to ``None`` instead of having values, as shown below:
 
-			::
+            ::
 
-				settings = None
+                settings = None
 
 
 **3. Initialize a ResourceManager object**
```

Comment:
I believe this section should also be removed since we already have one and this one contains outdated information

Reply:
This is a separate subsection of a larger `Setting up a ResourceManager object (legacy)` section, which describes a legacy way to connect.